jax_test3.py

```diff
@@ -8,34 +8,50 @@
 
 def E_calc(x_a, y_a, z_a, x_b, y_b, z_b, c_a, c_b):
     d = jnp.sqrt((x_a - x_b) ** 2 + (y_a - y_b) ** 2 + (z_a - z_b) ** 2)
-    E = (c_a * c_b) / d
-    return E
+    energy = (c_a * c_b) / d
+    return energy
```

Comment:
This needs to be implemented to accept a matrix of coordinates and a vector of charges. We're not calculating the correct energy/gradient/Hessian otherwise.
The inner loop could look something like:
```python
energy = 0.0
natoms = charges.size
for A in range(natoms):
    for B in range(A):
         d = jnp.linalg.norm(coordinates[A, :] - coordinates[B, :])
         energy += charges[A] * charges[B] / d
```